### ISSUE TITLE
MQTT reconnect - verify/add functionality

### ISSUE THREAD

**Issue by lkankowski** [lkankowski/arduino-multi-relay2#23]
1. Verify if there is no reconnect functionality in MySensors
2. Add reconnect functionality

**Comment by chytreg** [comment]
Ad 1) https://github.com/mysensors/MySensors/blob/development/core/MyGatewayTransportMQTTClient.cpp#L145-L165
Ad 2) I don't think it's needed  ^

I would ask another question. Does the system work properly when there is no connection to the MQTT server?

**Comment by lkankowski** [comment]
Yes, the system will continue to function normally, although I am concerned about the "delay" commands in the MySensors library.
